src/glossary.md

```diff
@@ -84,6 +81,19 @@ one of the design goals of a blockchain is achieving accountability of the
 parties
 (e.g., internal and external auditors, regulators and end users of the system).
 
+## Blockchain Explorer
+
+An [application][explorer] for monitoring blocks and transactions in Exonum
```

Comment:
`application` is not suitable definition I guess. An application it's something holistic. But blockchain explorer is a part of exonum framework that provides some functionality like monitoring blocks and transactions. Therefore I'd name it module or even REST API for monitoring blocks and transactions. 
```A module that provides REST APT for monitoring blocks and transactions...``` 
maybe smth like this.